test/libsolidity/syntaxTests/structs/recursion/return_recursive_structs2.sol

```diff
@@ -1,7 +1,9 @@
+pragma experimental ABIEncoderV2;
+
 contract C {
     struct S { uint a; S[2][] sub; }
     function f() public pure returns (uint, S memory) {
     }
 }
```

Comment:
Please consider adding test cases that define an array of mappings as function arguments such as

```
function f(mapping(uint => uint)[] memory x) public pure {}
```

perhaps even in libraries and interfaces?

Test cases I could think of

```
pragma experimental ABIEncoderV2;
library L {
        struct S { mapping(uint => uint) m; }
        function f(S memory a) external pure returns (S memory) {}
}
```

```
pragma experimental ABIEncoderV2;
library L {
        function f(mapping(uint => uint)[2] memory a) external pure returns (mapping(uint => uint)[2] memory) {}
}
```